lib/lndclient/LndClient.ts

```diff
@@ -382,19 +395,76 @@ class LndClient extends SwapClient {
   /**
    * Connects to another lnd node.
    */
-  public connectPeer = (pubkey: string, host: string, port: number): Promise<lndrpc.ConnectPeerResponse> => {
+  public connectPeer = (pubkey: string, host: string): Promise<lndrpc.ConnectPeerResponse> => {
     const request = new lndrpc.ConnectPeerRequest();
     const address = new lndrpc.LightningAddress();
-    address.setHost(`${host}:${port}`);
+    address.setHost(host);
     address.setPubkey(pubkey);
     request.setAddr(address);
     return this.unaryCall<lndrpc.ConnectPeerRequest, lndrpc.ConnectPeerResponse>('connectPeer', request);
   }
 
+  /**
+   * Opens a channel given peerPubKey and amount.
+   */
+  public openChannel = async (
+    peerPubKey: string,
+    amount: number,
+    currency: string,
```

Comment:
It's a little weird to me to pass `currency` in as a parameter here. It seems like the logic to use the right lnd client for the currency should be handled outside of this class. Technically we could have this currency check in other functions as well, but I don't think that's the best approach.

Edit: I see that this is mainly used for Raiden and we want to keep a common interface between that and lnd, so I see the reasoning here. Still, I might prefer making `currency` an optional parameter for the `openChannel` method and just ignore it for the lnd client.

Reply:
Agreed. I changed the function to use destructuring and also changed `peerPubKey` to `peerIdentifier` because in the context of Raiden we don't have `peerPubKey`.